pkg/deploy/generator/resources.go

```diff
@@ -697,7 +697,10 @@ rpm --import https://dl.fedoraproject.org/pub/epel/RPM-GPG-KEY-EPEL-7
 rpm --import https://packages.microsoft.com/keys/microsoft.asc
 rpm --import https://packages.fluentbit.io/fluentbit.key
 
-yum -y install https://dl.fedoraproject.org/pub/epel/epel-release-latest-7.noarch.rpm
+for attempt in {1..5}; do
```

Comment:
`{1..5}` - TIL

Reply:
I thought that was cool, too

Reply:
@asalkeld did this first in v3 :D I just copy-pasted :dagger: